library/Rules/Even.php

```diff
@@ -13,10 +13,24 @@
 
 namespace Respect\Validation\Rules;
 
-class Even extends AbstractRule
+/**
+ * Validates an even number.
+ *
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Jean Pimentel <jeanfap@gmail.com>
+ * @author Paul karikari <paulkarikari1@gmail.com>
+ */
+final class Even extends AbstractRule
 {
+    /**
+     * {@inheritdoc}
+     */
     public function validate($input): bool
     {
+        if (!is_numeric($input)) {
```

Comment:
I realized that even numbers must be integers. A validation using `filter_var($input, FILTER_VALIDATE_INT)` must be added